elements/title.html

```diff
@@ -39,7 +39,7 @@
   <pre class="code-example">
     <code>
       &lt;head&gt;
-        &lt;h1&gt;<strong>&lt;style&gt;</strong>This content is styled in CSS<strong>&lt;style&gt;</strong>&lt;h1&gt;
+        &lt;h1&gt;<strong>&lt;style&gt;</strong>This content is styled in CSS<strong>&lt;/style&gt;</strong>&lt;/h1&gt;
```

Comment:
the word `style` is used instead of `title` in this code example

![image](https://cloud.githubusercontent.com/assets/9439053/12462620/e0342d1a-bf84-11e5-8ab9-03f2925f20f7.png)